server/sonar-server/src/main/java/org/sonar/server/measure/ws/ComponentAction.java

```diff
@@ -242,8 +249,8 @@ private static void addBestValuesToMeasures(List<MeasureDto> measures, Component
 
   private static ComponentWsRequest toComponentWsRequest(Request request) {
     ComponentWsRequest componentWsRequest = new ComponentWsRequest()
-      .setComponentId(request.param(PARAM_COMPONENT_ID))
-      .setComponentKey(request.param(PARAM_COMPONENT_KEY))
+      .setComponentId(request.param(DEPRECATED_PARAM_COMPONENT_ID))
+      .setComponentKey(request.param(PARAM_COMPONENT))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "setComponentKey"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)